packages/appstore/src/components/modals/deriv-go-modal/deriv-go-modal-desktop/deriv-go-modal-desktop.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import QRCode from 'qrcode.react';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import { Modal, Text, Icon } from '@deriv/components';
+import { getUrlBase } from '@deriv/shared';
+import {
+    DERIV_PLATFORM_NAMES,
+    getPlatformDerivGoDownloadLink,
+    MOBILE_PLATFORMS,
+    DERIVGO_QRCODE_APP_URL,
+} from 'Constants/platform-config';
+import TradingPlatformIconProps from 'Assets/svgs/trading-platform';
+import './deriv-go-modal-desktop.scss';
+
+export const DerivGoModalDesktop = observer(() => {
+    const { traders_hub, ui } = useStore();
+    const { is_deriv_go_modal_visible, setIsDerivGoModalVisible } = traders_hub;
+    const { is_dark_mode_on } = ui;
+
+    const handleModalClose = () => {
+        setIsDerivGoModalVisible(false);
+    };
+
+    return (
+        <Modal
+            is_open={is_deriv_go_modal_visible}
+            toggleModal={handleModalClose}
+            width='701px'
```

Comment:
```suggestion
            width='700px'
```

Reply:
@mahdiyeh-deriv sure, I can change the width, no problem, but I followed the design (image attached)

<img width="486" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/2baf626b-6399-4ff8-8980-9be98b12a7ab">